cmd/markdown-link-check.sample.yml

```diff
@@ -2,3 +2,9 @@ ignore:
   - ^ftp:\/\/
   - ^http:\/\/
   - ^https:\/\/
+
+provider:
+  github:
+    nitro:
```

Comment:
Maybe to make this generic change it to 'organization'.

Reply:
Hmmm change what? The `owner`?